CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Prism is correctly handling the `csv` collection format argument property in OAS2 documents #577
 - Prism is correctly returning the response when the request has `*/*` as Accept header #578
 - Prism is correctly returning a single root node with the payload for XML data #578
+- Prism is correctly serializing application/json responses
```

Comment:
These changelog messages are not as effective at communicating what happened.

"Prism is" we dont need this because we know its Prism
"correctly" we know we fixed something this in in the fixed section

What was fixed? What was it doing before? What is the scenario in which application/json scenarios were broken?

People read changelogs to find out if a specific situation that they might have noticed has been fixed, but this is not giving them that information. It's just kinda vaguely saying "something about json works better than before"

Reply:
The correct changeling here would be a line explaining that Prism is now giving higher priority to `application/json` Accept content type, and it's not a "fallback" format anymore.

Reply:
Moreover this is not in the right section, it should go in the `Unreleased` part